lib/pf/api.pm

```diff
@@ -1156,6 +1149,35 @@ sub radius_rest_switch_authorize :Public :RestPath(/radius/rest/switch/authorize
 
     return \%mapped_return;
 }
+
+sub handle_accounting_metadata : Public {
+    my ($class, %RAD_REQUEST) = @_;
+    my $logger = pf::log::get_logger();
+    $logger->debug("Entering handling of accounting metadata");
+    my $client = pf::client::getClient();
```

Comment:
Since this call is executed in the webservices, calling getClient will return the local one meaning it will be executed in the same process which is desirable.

Otherwise the webservices call will queue 2 tasks while its already being executed async